auto_landing/src/detect_platform.cpp

```diff
@@ -0,0 +1,159 @@
+#include <auto_landing/detect_platform.hpp>
+
+namespace ariitk::detect
+{
```

Comment:
Refer https://github.com/AerialRobotics-IITK/ariitk_auto_landing/pull/11/files#r411117212
for this file as well